schema/contact.json

```diff
@@ -32,6 +32,16 @@
         ]
       }
     },
+   "externalIdentifiers": {
```

Comment:
@dwalt I think we should use "externalIdentifier" (singular) to remain consistent with the existing definitions.

Reply:
I agree with the change.